### PR TITLE
fix(workflow): wrong inline comments processing

### PR DESCRIPTION
**Pull request by k-dovgan**

### CODE REVIEW

File: .github/workflows/telegram-bot.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ on:
     types: [completed]
   pull_request_review:
     types: [submitted, edited, dismissed]
+  pull_request_review_comment:
+    types: [created]
```
Comment (k-dovgan) [inline .github/workflows/telegram-bot.yml:13]: Second test.

**Comment by k-dovgan** [comment]
Other test.

File: .github/workflows/telegram-bot.yml [hunk 2]
```diff
@@ -48,14 +49,20 @@ jobs:
 
         elif [[ ! -z "${{ github.event.comment }}" ]]; then
           ESCAPED_TEXT=`echo -e "${{ env.COMMENT_BODY }}"| sed 's/\&/\&amp;/g' | sed 's/</\&lt;/g' | sed 's/>/\&gt;/g'`
-          TEXT=`echo -e "<b>${{ env.COMMENT_AUTHOR }}</b> posted the following comment to issue <a href=\"${{ env.COMMENT_URL }}\">#${{ env.COMMENT_NUMBER }}</a>:\n<i>$ESCAPED_TEXT</i>"`
+          if [[ ! -z "${{ github.event.pull_request }}" ]]; then
```
Comment (i-keliukh) [inline .github/workflows/telegram-bot.yml:52]: Test 1

**Comment by i-keliukh** [comment]
Test 3

File: .github/workflows/telegram-bot.yml [hunk 3]
```diff
@@ -80,3 +80,10 @@ jobs:
 
       env:
         URL: https://api.telegram.org/bot${{ secrets.TELEGRAM_BOT_TOKEN }}/sendMessage
+
+  info:
+    name: print payload
```
Comment (i-keliukh) [inline .github/workflows/telegram-bot.yml:85]: Test 4

**Review by i-keliukh — CHANGES_REQUESTED** [review]
Test 5

**Review by i-keliukh — COMMENTED** [review]
Test 6

File: .github/workflows/telegram-bot.yml [hunk 4]
```diff
@@ -9,6 +9,8 @@ on:
     types: [completed]
   pull_request_review:
     types: [submitted, edited, dismissed]
+  pull_request_review_comment:
```
Comment (k-dovgan) [inline .github/workflows/telegram-bot.yml:12]: Try adding path.

**Review by k-dovgan — COMMENTED** [review]
This is review comment.

**Review by miltolstoy — APPROVED** [review]

**Review by i-keliukh — APPROVED** [review]

**Pull request merged**